transactor/src/test/scala/io/mediachain/transactor/DummyContext.scala

```diff
@@ -0,0 +1,44 @@
+package io.mediachain.transactor
+
+import io.atomix.copycat.client.CopycatClient
+import io.atomix.copycat.server.CopycatServer
+import io.atomix.catalyst.transport.Address
+
+case class DummyContext(
+  server: CopycatServer, 
+  client: CopycatClient, 
+  store: Types.Datastore,
+  logdir: String
+)
+
+object DummyContext {
+  def setup(srvaddr: String, logdir: String) = {
+    println("*** SETUP DUMMY COPYCAT CONTEXT")
+    setupLogdir(logdir)
+    val address = new Address(srvaddr)
+    val store = new Dummies.DummyStore
+    val server = Copycat.Server.build(address, logdir, store)
+    server.bootstrap().join()
```

Comment:
no timeout argument in join, we have to bite it -- but I think it fails eventually anyway by contract.